modules/data.tf

```diff
@@ -0,0 +1,19 @@
+data "aws_subnet" "public_a" {
```

Comment:
creeaza un folder in modules, denumestel cu numele modulului si muta totul acolo